### PR TITLE
fix(jest-enviroment): add EdgeRuntime global

### LINKED ISSUE #75: @edge-runtime/jest-environment does not define EdgeRuntime global
It seems as if the `@edge-runtime/jest-environment` package does not define the `EdgeRuntime` global.

This prevents me from addressing the runtime in unit tests, which should normally work using

```js
if (typeof EdgeRuntime !== 'string') {
  // dead-code elimination is enabled for the code inside this block
}
```

https://edge-runtime.vercel.sh/features/available-apis#addressing-the-runtime

### PR DESCRIPTION
**Pull request by Schniz**
This fixes #75

### CODE REVIEW

**Comment by Schniz** [comment]
@kikobeats do you think we should add the `EdgeRuntime` global to EdgeVM instead of repeating the declaration in multiple places?

**Comment by feugy** [comment]
It'll be better defined in the EdgeVM class.

**Comment by Kikobeats** [comment]
@Schniz @feugy yes, better if we can centralize to write it just in one place 🙂

File: packages/jest-environment/src/index.ts [hunk 1]
```diff
@@ -20,6 +20,9 @@ export = class EdgeEnvironment implements JestEnvironment<number> {
       extend: (context) => {
         context.global = context
         context.Buffer = Buffer
+        Object.defineProperty(context, 'EdgeRuntime', {
+          value: 'edge-runtime',
+        })
```
Comment (Schniz) [inline packages/jest-environment/src/index.ts:25]: whoops this needs to be removed

Comment (Schniz) [reply packages/jest-environment/src/index.ts:25]:
```suggestion
```

File: packages/vm/src/edge-vm.ts [hunk 2]
```diff
@@ -89,6 +90,7 @@ function addPrimitives(context: VMContext) {
   defineProperty(context, 'clearTimeout', { value: clearTimeout })
   defineProperty(context, 'setInterval', { value: setInterval })
   defineProperty(context, 'setTimeout', { value: setTimeout })
+  defineProperty(context, 'EdgeRuntime', { value: 'edge-runtime' })
```
Comment (feugy) [inline packages/vm/src/edge-vm.ts:93]: **suggestion:** maybe we could make it non-writable?

By default `defineProperty` creates writable properties (despite not being configurable, their value could still be [changed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#configurable)).

**Review by feugy — APPROVED** [review]
🚢  it!

My suggestion is not a blocker.

File: packages/vm/src/edge-vm.ts (hunk 2 above)
Comment (Schniz) [reply packages/vm/src/edge-vm.ts:93]: I think it's fine for it to be writable. Maybe someone would want to extend it with a custom flavor and set it to `custom-edge-runtime`. what do you think?

Comment (feugy) [reply packages/vm/src/edge-vm.ts:93]: well, it's JS. We should not be too prescriptive! 😅 
Let's go with writable.

**Pull request merged**